src/v/cluster/tests/partition_moving_test.cc

```diff
@@ -786,11 +818,15 @@ FIXTURE_TEST(
 
                 wait_for_replica_set_partitions(60s, ntp, r);
 
-                validate_replicas_recovery(20s, ntp, r, reference_batches);
+                validate_replicas_recovery(60s, ntp, r, reference_batches);
```

Comment:
1 min seems like a lot to me for a test. But ok. Do you expect this to take up to 1 min? If so we should write a comment to why.